libgalois/include/katana/PropertyGraph.h

```diff
@@ -246,10 +246,25 @@ class KATANA_EXPORT PropertyGraph {
   /// A map from the node EntityTypeID to its type name if it is an atomic type
   /// (that does not intersect any other atomic type)
   EntityTypeIDToAtomicTypeNameMap node_atomic_entity_type_id_to_type_name_;
+
   /// A map from the edge EntityTypeID to its type name if it is an atomic type
   /// (that does not intersect any other atomic type)
   EntityTypeIDToAtomicTypeNameMap edge_atomic_entity_type_id_to_type_name_;
 
+  /// A vector of pairs representing flattened mapping between the node entity ids
+  tsuba::EntityTypeIDToEntityTypeIDPairs
```

Comment:
I'm assuming you guys will remove these pair vectors from being members and make them temporary variables instead? We don't need to keep them around.